na-mainloop/Cargo.toml

```diff
@@ -9,6 +9,9 @@ edition = "2021"
 log = "0.4"
 android_logger = "0.11.0"
 android-activity = { version = "0.4", features = [ "native-activity" ] }
+#android-activity = { path = "../../android-activity/android-activity", features = [ "native-activity" ] }
```

Comment:
Only noticing this now during a rebase, what is it useful for?

Reply:
A slight convenience for me, for testing at time but it would probably be better to have a commented out `[patches]` section for something like that.

It's fine if you drop it.